packages/ui/tsconfig.json

```
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "strict": false
   },
-  "include": ["src/**/*.ts", "src/**/*.tsx"],
+  "include": ["src/**/*.ts", "src/**/*.tsx", "benchmarks/**/*.ts"],
```

Think `benchmarks` was just for testing?
```suggestion
  "include": ["src/**/*.ts", "src/**/*.tsx"],
```